test/integration/targets/vmware_guest_facts/tasks/main.yml

```diff
@@ -1,6 +1,12 @@
 # Test code for the vmware_guest_facts module.
 # Copyright: (c) 2017, Abhijeet Kasurde <akasurde@redhat.com>
+# Copyright (C) 2018 James E. King III (@jeking3) <jking@apache.org>
 # GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+---
+- name: Install required library updates
+  pip:
```

Comment:
Instead of installing the requirements here, add a version constraint for all the tests to use:

https://github.com/ansible/ansible/blob/devel/test/runner/requirements/constraints.txt

Reply:
Thanks, I changed it.